app/Providers/EventServiceProvider.php

```diff
@@ -25,14 +25,23 @@ class EventServiceProvider extends ServiceProvider
     ];
 
     /**
-     * Register any events for your application.
+     * The event to listener mappings for the application.
      *
-     * @return void
+     * @var array<class-string, array<int, class-string>>
```

Comment:
What does this @ var do?

Reply:
Also upstream. I think this silences the output.

Reply:
I feel like something got a bit mixed up here. Some example EventServiceProvider stuff I found online looked like this:
`class EventServiceProvider extends ServiceProvider
{
    /**
     * The event to listener mappings for the application.
     *
     * @var array<class-string, array<int, class-string>>
     */
    protected $listen = [
        Registered::class => [
            SendEmailVerificationNotification::class,
        ],
    ];

    /**
     * Register any events for your application.
     */
    public function boot(): void
    {
        Ad::observe(AdObserver::class);
    }

    /**
     * Determine if events and listeners should be automatically discovered.
     */
    public function shouldDiscoverEvents(): bool
    {
        return false;
    }
}`

So I think either the $listen property should be added + the boot() comment should be re-added, or: leave out the $listen and just correct the comment to actually describe the boot() method.